README.md

```diff
@@ -1,15 +1,15 @@
 # bootcamp
-A Minimal social networking application using Flask
+A Minimal social networking application using Flask Applicaion
 
 ## Setup
 
 1. **Clone this Repo**
 
     ``` 
-    git clone https://github.com/SriNandan33/bootcamp.git 
+    git clone https://github.com/devkushal23/bootcamp.git 
```

Comment:
You should leave this link as the original. The original project is from @SriNandan33

Reply:
Oh Sorry I made that correction by mistake